File: src/main/java/dark/leech/text/models/Trash.java
package dark.leech.text.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/** Code by Darkrai on 8/21/2016. */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Trash {
    private String src = "";
    private String to = "";
    private String tip = "";
    private boolean replace = true;

    public String getTo() {
        if (to == null) to = "";
        return to;
    }
}

File: engine/src/commonMain/kotlin/dev/haipham22/leechtext/models/Trash.kt
package dev.haipham22.leechtext.models

import kotlinx.serialization.Serializable

/** Rule regex thay thế text (port từ models/Trash.java). */
@Serializable
data class Trash(
    var src: String? = "",
    var to: String? = "",
    var tip: String? = "",
    var replace: Boolean = true,
)
